ouroboros-network-framework/src/Simulation/Network/Snocket.hs

```diff
@@ -23,6 +23,7 @@ module Simulation.Network.Snocket
     withSnocket
   , ResourceException (..)
   , SnocketTrace (..)
+  , TimeoutDetail (..)
```

Comment:
This seems to belong to another commit. `snocket: better mask/unmask snocket and socket resources`.